File: TOI-Zero_68/A1/025/025.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {
    char s[10];
    scanf("%s", s);
    
    // Convert to uppercase
    for(int i = 0; s[i]; i++) {
        s[i] = toupper(s[i]);
    }
    
    char a[3], b[2];
    if(isdigit(s[0])) {
        if(strlen(s) >= 3 && s[0] == '1' && s[1] == '0') {
            a[0] = '1';
            a[1] = '0';
            a[2] = '\0';
            strcpy(b, &s[2]);
        } else {
            a[0] = s[0];
            a[1] = '\0';
            strcpy(b, &s[1]);
        }
    } else {
        a[0] = s[0];
        a[1] = '\0';
        strcpy(b, &s[1]);
    }
    
    const char* c;
    if(strcmp(a, "A") == 0) {
        c = "ace";
    } else if(strcmp(a, "J") == 0) {
        c = "jack";
    } else if(strcmp(a, "Q") == 0) {
        c = "queen";
    } else if(strcmp(a, "K") == 0) {
        c = "king";
    } else {
        c = a;
    }
    
    const char* d;
    if(strcmp(b, "D") == 0) {
        d = "diamonds";
    } else if(strcmp(b, "H") == 0) {
        d = "hearts";
    } else if(strcmp(b, "S") == 0) {
        d = "spades";
    } else if(strcmp(b, "C") == 0) {
        d = "clubs";
    }
    
    printf("%s of %s\n", c, d);
    
    return 0;
} 

File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A1/025/025.py
s = input().upper()
if s[0].isdigit():
    if len(s) >= 3 and s[0] == '1' and s[1] == '0':
        a = "10"
        b = s[2:]
    else:
        a = s[0]
        b = s[1:]
else:
    a = s[0]
    b = s[1:]

if a == "A":
    c = "ace"
elif a == "J":
    c = "jack"
elif a == "Q":
    c = "queen"
elif a == "K":
    c = "king"
else:
    c = a

if b == "D":
    d = "diamonds"
elif b == "H":
    d = "hearts"
elif b == "S":
    d = "spades"
elif b == "C":
    d = "clubs"

print(f"{c} of {d}")

